Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -30,6 +30,7 @@
 #include <WebCore/PseudoElementIdentifier.h>
 #include <WebCore/StylePrimitiveNumeric+Forward.h>
 #include <WebCore/StyleTextDecorationLine.h>
+#include <WebCore/StyleWebKitBorderSpacing.h>
```

Comment:
Please don't add new includes to RenderStyle.h. Forward declarations should work fine.